test/functional/specs/C148846.js

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { t, ClientFunction } from "testcafe";
+import createNetworkLogger from "../helpers/networkLogger";
+import fixtureFactory from "../helpers/fixtureFactory";
+import configureAlloyInstance from "../helpers/configureAlloyInstance";
+import {
+  compose,
+  edgeDomainFirstParty,
+  orgMainConfigMain
+} from "../helpers/constants/configParts";
+import { domain } from "../helpers/edgeInfo";
+import getResponseBody from "../helpers/networkLogger/getResponseBody";
+import createResponse from "../../../src/core/createResponse";
+
+const executeEventCommand = ClientFunction(() => {
+  return window.alloy("sendEvent");
+});
+
+const demdexHostRegex = /\.demdex\.net/;
+
+const getHostFor = requestLogger => requestLogger.request.headers.host;
+
+const browsersSupportingThirdPartyCookiesByDefault = [
+  "Chrome",
+  "Chromium",
+  "Microsoft Edge",
+  "Internet Explorer"
+];
+
+const identityCookieName = "kndctr_334F60F35E1597910A495EC2_AdobeOrg_identity";
+
+const areThirdPartyCookiesSupportedByBrowserByDefault = () =>
+  browsersSupportingThirdPartyCookiesByDefault.indexOf(t.browser.name) !== -1;
```

Comment:
It might be worth moving this (and the list of browsers) into a helper, since it's also used in another test and potentially future tests.

Reply:
Good idea, will do.